App.tsx

```diff
@@ -17,33 +17,102 @@ import {
   Text,
   StatusBar,
   Button,
-} from 'react-native';
+  Keyboard,
+} from "react-native";
+import { TouchableWithoutFeedback } from "react-native";
 
 import {
   Header,
   LearnMoreLinks,
   Colors,
   DebugInstructions,
   ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
-import WelcomeScreen from './WelcomeScreen';
+} from "react-native/Libraries/NewAppScreen";
+import WelcomeScreen from "./WelcomeScreen";
+import SignUp from "./SignUp";
 
-declare var global: {HermesInternal: null | {}};
+declare var global: { HermesInternal: null | {} };
 
 const App = () => {
   return (
-    <>
-      <WelcomeScreen />
-    </>
+    <TouchableWithoutFeedback
```

Comment:
What's the reason behind using <TouchableWithoutFeedback /> here?

Reply:
So that i can call the function onPress={Keyboard.dismiss}.
It's not working with other components.